### DISCUSSION TITLE
Multiple Google Takeouts with shared albums?

### DISCUSSION THREAD

**Discussion by AgentScrubbles** [simulot/immich-go#198] [Q&A]
My SO and I have obviously a ton of shared albums and shared photos, does your import automatically merge photos and albums, detect duplicates?  One way to find out obviously, I may try it in a bit regardless.  Not afraid of a bit of manual work either, but I'm hoping to not have hundreds of duplicates I have to parse through.

**Comment by simulot** [comment]
You have have ask separate takeout for you SO (what is SO). But you'll get shared photos twice.
Immich (the application) doesn't handle yet the duplicate control across multiple accounts. Immich-go can't do best at the moment.

On top of this, you may have activated the google compression for the storage.

The is an immich-go options to not import shared photos, another to place them into a dedicated album, that can be shared as well.
But you'll end up with duplicates between users.

Have you a suggestion on the way to manage the problem?
